backend/apps/volontulo/urls.py

```diff
@@ -50,7 +50,21 @@
         api_views.password_reset_confirm,
         name='password_reset_confirm'
     ),
-    url(r'^api/messages/$', api_views.messages_view, name='messages'),
+    url(
+        r'^api/messages/$',
+        api_views.messages_view,
+        name='messages'
+    ),
+    url(
+        r'^api/register',
```

Comment:
whereas such an endpoint will obviously work fine, it's not really RESTful. Since the plan is to have REST API, we should rather whenever we can operate on resources.

I think i.e. POST request to `api/users` would better fit REST design. I don't think it's the top concern of this app at the moment, so feel free to discard this remark.

Reply:
I would prefer to to so massive change in other pull request - this one is limited to moving register view from Django to Angular.

BTW @jacekkalbarczyk - we decided to add trailing slash for API endpoint (so url should looks like: `'^api/register/$'`)